src/main/java/tool/arenasizer/ArenaInfoPanel.java

```diff
@@ -0,0 +1,174 @@
+        futureCapacityPanel.labelPercentageVipBoxes.setText(stadium.getZukunftLogen().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+        futureCapacityPanel.labelPercentageTotal.setText(stadium.getZukunftGesamtgroesse().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+
+        futureCapacityPanel.label2.setText(Optional.ofNullable(stadium.getExpansionDate())
+                .map(expansionDate -> nextDay(expansionDate).toLocaleDate())
+                .orElse(EMPTY));
+
+        setTranslation();
+    }
+
+    private static HODateTime nextDay(HODateTime hoDateTime) {
+        var localDate = hoDateTime.getLocalDateTime().toLocalDate();
+        var nextDay = localDate.plusDays(1);
+        var instant = Instant.from(nextDay.atStartOfDay(HODateTime.DEFAULT_TIMEZONE));
+        return new HODateTime(instant);
+    }
+
+    private void setTranslation() {
+        currentCapacityPanel.setTranslation();
+        currentCapacityPanel.setTitle(getLanguageString("ArenaInfoPanel.current"));
+        currentCapacityPanel.label1.setText(getLanguageString("ArenaInfoPanel.last_improvement"));
+
+        expandedCapacityPanel.setTranslation();
+        expandedCapacityPanel.setTitle(getLanguageString("ArenaInfoPanel.expansion"));
+        expandedCapacityPanel.labelPercentage.setText(getLanguageString("ArenaInfoPanel.change_in_percent"));
+        expandedCapacityPanel.label1.setText(getLanguageString("ArenaInfoPanel.completion"));
+
+        futureCapacityPanel.setTranslation();
+        futureCapacityPanel.setTitle(getLanguageString("ArenaInfoPanel.future"));
+        futureCapacityPanel.label1.setText(getLanguageString("ArenaInfoPanel.available"));
+    }
+
+    private void updateExpansionFinishedLabel() {
+        final JLabel labelExpansionFinished = expandedCapacityPanel.label3;
+        final Stadium stadium = HOVerwaltung.instance().getModel().getStadium();
+        Optional.ofNullable(stadium).map(Stadium::getExpansionDate).ifPresentOrElse(expansionDate -> {
+                    final String text = toDurationString(expansionDate);
+                    final String toolTipText = String.format(getLanguageString("ArenaInfoPanel.finished_in_n_days_format"), calculateDaysToNow(expansionDate, 1));
+                    labelExpansionFinished.setText(text);
+                    labelExpansionFinished.setToolTipText(toolTipText);
+                },
+                () -> {
+                    labelExpansionFinished.setToolTipText(null);
+                    labelExpansionFinished.setText(EMPTY);
+                }
+        );
+    }
+
+    private static String getLanguageString(String key) {
+        return HOVerwaltung.instance().getLanguageString(key);
+    }
+
+    private static BigDecimal calculateDaysToNow(HODateTime hoDateTime, int scale) {
```

Comment:
Could this somehow be pushed into `HODataTime`?  Or is there an equivalent method there, maybe?

Reply:
Duration = HODateTime.between(HODateTime from, HODateTime to)

Reply:
Yes.

Committed as 'Review Finding 1'

Reply:
I saw your new comment too late.
Will be fixed tomorrow.

Reply:
I decided to name it `daysBetween`, `daysToNow` and `daysToNow`.